server/server.go

```diff
@@ -26,21 +26,6 @@ type ServerConfig struct {
 	OfflineStatus infrared.StatusConfig `json:"offlineStatus"`
 }
 
-type LoginServer interface {
-	Login(conn connection.LoginConn) error
-}
-
-type StatusServer interface {
-	Status(conn connection.StatusConn) protocol.Packet
-}
-
-// Server will act as abstraction layer between connection and proxy ...?
-type Server interface {
-	LoginServer
-	StatusServer
-	Start()
-}
-
 type MCServer struct {
```

Comment:
We could maybe call this JavaServer

Reply:
I dont think its necessary worth it at the moment to think about naming variables in such a way that they dont have to be renamed in the future when we implement another version of a server? 
Im not even sure or we should call it a server, like we know that there should be some minecraft server behind it which is most likely going to be in java but what if its written in another language? Its unsure whether there are 3 proxies between this server struct and the server or that it will call towards the server directly.  What if it calls a bungeecord server?